[Blog] PulumiUP - the Event for Cloud Engineers (#51)
pulumi/pulumi-hugo: themes/default/content/blog/pulumiup/index.md

---

> title: "PulumiUP - the Event for Cloud Engineers"

Reviewer: ```suggestion
title: "PulumiUP: the Event for Cloud Engineers"
```

date: 2021-04-14

> meta_desc: "PulumiUP is two hour keynote with cloud engineering industry leaders Adrian Cockcroft, VP of Sustainability Engineering at Amazon, Justin Fitzhugh, VP of Cloud Engineering at Snowflake, Kieth Redmond, VP of SaasS Engineering at Fenergo."

Reviewer: ```suggestion
meta_desc: "PulumiUP is a two hour keynote with cloud engineering industry leaders Adrian Cockcroft, VP of Sustainability Engineering at Amazon, Justin Fitzhugh, VP of Cloud Engineering at Snowflake, Keith Redmond, VP of SaaS Engineering at Fenergo."
```

meta_image: pulumiup.png
authors:
    - wendy-smith
tags:

>     - cloud engineering

Reviewer: ```suggestion
    - cloud-engineering
```

Not sure if we have a standard for tags with spaces or not?

    - PulumiUP
---

> My professional background has included nearly ten years of managing field events and user conferences. I never thought I would say this, but I miss traveling. I even missed Vegas and reInvent this year. I miss connecting with customers and advocates in our communities. I wish we could all be looking forward to getting together in person in Seattle or Austin or insert any city here. As the year continued, it became clear we were not going back to in-person events anytime soon, and everyone in the industry pivoted to virtual programs while video conferencing became an all-day activity.

Reviewer: ```suggestion
My professional background has included nearly ten years of managing field events and user conferences. I never thought I would say this, but I miss traveling. I even missed Vegas and AWS reInvent this year. I miss connecting with customers and advocates in our communities. I wish we could all be looking forward to getting together in person in Seattle or Austin or _insert any city here_. As the year continued, it became clear we were not going back to in-person events anytime soon, and everyone in the industry pivoted to virtual programs while video conferencing became an all-day activity.
```

<!--more-->

> We spent a lot of time considering how to best engage and share announcements with our community and encourage the cloud engineering community's growth. All of us here at Pulumi recognize how valuable your time is and how much time in the last year we spent on virtual event platforms. With that in mind, we made a deliberate decision to avoid any all-day or multi-day programming. Instead, we are hosting a short 2-hour keynote that we think you will find insightful and valuable - best practices and real-life situations that you can learn from and apply to your approach to building, delivering, and managing modern applications.

Reviewer: ```suggestion
We spent a lot of time considering how to best engage and share announcements with our community and encourage the cloud engineering community's growth. All of us here at Pulumi recognize how valuable your time is and how much time we've all spent on virtual event platforms in the last year. With that in mind, we made a deliberate decision to avoid any all-day or multi-day programming. Instead, we are hosting a short 2-hour keynote that we think you will find insightful and valuable, with best practices and real-life situations that you can learn from and apply to your approach to building, delivering, and managing modern cloud applications.
```

| **Speakers** | **Title and Company** | **Subject** |
| - | - |- |
| Joe Duffy | Co-Founder and CEO, Pulumi | **The Modern Cloud Era: All Software is Cloud Software** |
|Adrian Cockcroft & Aaron Kao                | VP of Sustainability Architecture, Amazon VP of Marketing, Pulumi | **Discussion: What’s different about building in a modern cloud application world** |
| Justin Fitzhugh | VP of Cloud Engineering, Snowflake | **Perspectives from Cloud Engineering Pioneers** |
| Keith Redmond | VP of SaaS Engineering, Fenergo | **Perspectives from Cloud Engineering Pioneers** |
| Luke Hoban | CTO, Pulumi | **Placing Cloud Engineering in the Hands of Every Developer** |
| Sophia Parafina | Technical Marketing Manager, Pulumi|
| Komal Ali | Software Engineer |

| **Panel: Hear from the Experts: Planning for the Future of Modern Cloud Applications** | **Title and Company** |
|- | - |
| Moderator: Kat Cosgrove | Developer Advocate, JFrog |
| Justin Fitzhugh | VP of Cloud Engineering, Snowflake |
| Charity Majors |  CTO/Co-Founder, Honeycomb |
| Dana Lawson |  VP of Engineering, GitHub  |

> Engineering leadership from Snowflake, Cloud Data Warehousing organization, and Fenergo, a client lifecycle management provider for the financial services industry, will share their perspectives and experiences practicing cloud engineering for the modern cloud era. Justin Fitzhugh, VP of Cloud Engineering at Snowflake, and Keith Redmond, VP of SaaS Engineering at Fenergo, will each share how they reduced costs while empowering every developer on their team.

Reviewer: ```suggestion
Engineering leadership from Snowflake, the cloud data warehousing organization, and Fenergo, a client lifecycle management provider for the financial services industry, will share their perspectives and experiences practicing cloud engineering for the modern cloud era. Justin Fitzhugh, VP of Cloud Engineering at Snowflake, and Keith Redmond, VP of SaaS Engineering at Fenergo, will each share how they reduced costs while empowering every developer on their team.
```
